railties/lib/rails/paths.rb

```diff
@@ -175,6 +175,12 @@ def to_ary
         @paths
       end
 
+      def expanded_paths
```

Comment:
Yeah, it is a red flag that we have to add this method. We can handle new files inside app without having to change `Paths`, so I believe we can use the same machinery to load new locales files.